lib/config.js

```diff
@@ -0,0 +1,42 @@
+class Config {
+  constructor ({ kms, dynamo }) {
+    this.kms = kms
+    this.configCache = new Map()
+  }
+
+  async decrypt (data) {
+    return this.kms.decrypt({
+      CiphertextBlob: Buffer.from(data, 'base64')
+    }).promise().then(decrypted => decrypted.Plaintext.toString())
+  }
+
+  async getAndCacheValue (configKey) {
+    if (this.configCache.has(configKey)) {
+      return this.configCache.get(configKey)
+    }
+    const configValue = await this.dynamo.getConfigValue(configKey)
+    this.configCache.set(configKey, configValue)
+
+    return configValue
+  }
+
+  async getIlpConnectorAddress () {
```

Comment:
This can just pull from process.env directly, I don't think we have a good reason to encrypt the connector address. Also it looks to be looking at MONGO_URI

Reply:
fixed - and better safe than sorry, not hard to keep it encrypted IMO